app/src/androidTest/java/de/test/antennapod/handler/FeedHandlerTest.java

```diff
@@ -66,6 +67,7 @@ private Feed runFeedTest(Feed feed, FeedGenerator g, String encoding, long flags
         Feed parsedFeed = new Feed(feed.getDownload_url(), feed.getLastUpdate());
         parsedFeed.setFile_url(file.getAbsolutePath());
         parsedFeed.setDownloaded(true);
+        parsedFeed.setImageUrl("http://example.com/picture");
```

Comment:
Shouldn't this be done by `handler.parseFeed(parsedFeed);` if everything works fine? I think we should rather fix the parser instead of hard-coding the test answer.